CHANGELOG.md

```diff
@@ -2,6 +2,79 @@
 
 ## Unreleased
 
+## v0.5.0 Beta
+
+## 🛑 Breaking changes 🛑
+
+- Enable "new-metrics" by default (#1148)
+- Remove `export_resource_labels` flag from Zipkin exporter (#1163)
+- Add config settings for Http Client, change Zipkin to use it. (#1186)
```

Comment:
This needs to tell what is changed in Zipkin config. If you check the PR testdata.yaml shows it: "url" is renamed to "endpoint".

>Add config settings for Http Client
This is not worth mentioning, it is an internal change.